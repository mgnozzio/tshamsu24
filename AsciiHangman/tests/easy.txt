easy
e
a
s
y
