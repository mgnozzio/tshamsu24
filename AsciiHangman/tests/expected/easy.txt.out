Welcome to the ASCII Version of Hangman!
For best results, please run this from the command line.
Please enter a secret word: 
















































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































































   ____  
  |    | 
  |      
  |      
  |      
__|__    
Puzzle to solve: _ _ _ _ 
Please guess a letter: E
Puzzle to solve: E _ _ _ 
Please guess a letter: A
Puzzle to solve: E A _ _ 
Please guess a letter: S
Puzzle to solve: E A S _ 
Please guess a letter: Y
Success!  Player 2 wins!
